<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeetingInfoWidget</class>
 <widget class="QWidget" name="MeetingInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="meeting_name">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>20</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gill Sans</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
                    background: #505050;
                    border: 2px solid #303030;
                    border-bottom-color: #303030; /* same as the pane color */
                    border-top-left-radius: 3px;
                    border-top-right-radius: 3px;
                    min-width: 14.4ex;
                    padding: 15px;
                    }
                </string>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="start_time">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gill Sans</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTimeEdit {
                    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                    stop: 0 #686868, stop: 0.4 #505050,
                    stop: 0.5 #686868, stop: 1.0 #505050);
                    border: 2px solid #303030;
                    border-bottom-color: #303030; /* same as the pane color */
                    border-top-left-radius: 3px;
                    border-top-right-radius: 3px;
                    min-width: 14.4ex;
                    padding: 5px;
                    }


                    QTimeEdit:up-button {

                    width: 15px;

                    border-left-width: 1px;
                    border-left-color: darkgray;
                    border-left-style: solid;
                    border-top-right-radius: 3px;
                    border-bottom-right-radius: 3px;

                    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                    stop: 0 #008000, stop: 0.4 #228B22,
                    stop: 0.5 #008000, stop: 1.0 #228B22);
                    border: 2px solid #303030;
                    border-bottom-color: #303030; /* same as the pane color */
                    border-top-left-radius: 3px;
                    border-top-right-radius: 3px;
                    min-width: 14.4ex;
                    padding: 5px;
                    }

                    QTimeEdit:down-button {

                    width: 15px;

                    border-left-width: 1px;
                    border-left-color: darkgray;
                    border-left-style: solid;
                    border-top-right-radius: 3px;
                    border-bottom-right-radius: 3px;

                    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                    stop: 0 #B22222, stop: 0.4 #FF0000,
                    stop: 0.5 #B22222, stop: 1.0 #FF0000);
                    border: 2px solid #303030;
                    border-bottom-color: #303030; /* same as the pane color */
                    border-top-left-radius: 3px;
                    border-top-right-radius: 3px;
                    min-width: 14.4ex;
                    padding: 15px;
                    }

                    QTimeEdit::down-button:hover {
                    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                    stop: 0 #484848, stop: 0.4 #585858,
                    stop: 0.5 #585858, stop: 1.0 #484848);
                    }

                    QTimeEdit::down-button:hover, QTimeEdit::up-button:hover
                    {
                    border-color: rgb(255, 68, 14); /* same as the pane color */
                    }

                    QTimeEdit::up-button:selected, QTimeEdit::up-button:hover {
                    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                    stop: 0 #484848, stop: 0.4 #585858,
                    stop: 0.5 #585858, stop: 1.0 #484848);
                    }


                </string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTimeEdit" name="end_time">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gill Sans</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTimeEdit {
                    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                    stop: 0 #686868, stop: 0.4 #505050,
                    stop: 0.5 #686868, stop: 1.0 #505050);
                    border: 2px solid #303030;
                    border-bottom-color: #303030; /* same as the pane color */
                    border-top-left-radius: 3px;
                    border-top-right-radius: 3px;
                    min-width: 14.4ex;
                    padding: 5px;
                    }


                    QTimeEdit:up-button {

                    width: 15px;

                    border-left-width: 1px;
                    border-left-color: darkgray;
                    border-left-style: solid;
                    border-top-right-radius: 3px;
                    border-bottom-right-radius: 3px;

                    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                    stop: 0 #008000, stop: 0.4 #228B22,
                    stop: 0.5 #008000, stop: 1.0 #228B22);
                    border: 2px solid #303030;
                    border-bottom-color: #303030; /* same as the pane color */
                    border-top-left-radius: 3px;
                    border-top-right-radius: 3px;
                    min-width: 14.4ex;
                    padding: 5px;
                    }

                    QTimeEdit:down-button {

                    width: 15px;

                    border-left-width: 1px;
                    border-left-color: darkgray;
                    border-left-style: solid;
                    border-top-right-radius: 3px;
                    border-bottom-right-radius: 3px;

                    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                    stop: 0 #B22222, stop: 0.4 #FF0000,
                    stop: 0.5 #B22222, stop: 1.0 #FF0000);
                    border: 2px solid #303030;
                    border-bottom-color: #303030; /* same as the pane color */
                    border-top-left-radius: 3px;
                    border-top-right-radius: 3px;
                    min-width: 14.4ex;
                    padding: 15px;
                    }

                    QTimeEdit::down-button:hover {
                    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                    stop: 0 #484848, stop: 0.4 #585858,
                    stop: 0.5 #585858, stop: 1.0 #484848);
                    }

                    QTimeEdit::down-button:hover, QTimeEdit::up-button:hover
                    {
                    border-color: rgb(255, 68, 14); /* same as the pane color */
                    }

                    QTimeEdit::up-button:selected, QTimeEdit::up-button:hover {
                    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                    stop: 0 #484848, stop: 0.4 #585858,
                    stop: 0.5 #585858, stop: 1.0 #484848);
                    }

                    QTimeEdit::up-arraow
                    {
                    border : 2px solid black;
                    background-color : lightgreen;
                    visable: true;
                    }

                </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::UpDownArrows</enum>
   </property>
   <property name="specialValueText">
    <string/>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="meeting_url">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>541</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gill Sans</family>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
                    background: #505050;
                    border: 2px solid #303030;
                    border-bottom-color: #303030; /* same as the pane color */
                    border-top-left-radius: 3px;
                    border-top-right-radius: 3px;
                    min-width: 14.4ex;
                    padding: 15px;
                    }
                </string>
   </property>
   <property name="maxLength">
    <number>200</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>URL</string>
   </property>
  </widget>
  <widget class="QLabel" name="start_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gill Sans</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Start TIme</string>
   </property>
  </widget>
  <widget class="QLabel" name="end_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gill Sans</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>End Time</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>120</y>
     <width>61</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Up</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>170</y>
     <width>60</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Down</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Up</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>25</y>
     <width>60</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Down</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>90</y>
     <width>251</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
                    background: #505050;
                    border: 2px solid #303030;
                    border-bottom-color: #303030; /* same as the pane color */
                    border-top-left-radius: 3px;
                    border-top-right-radius: 3px;
                    min-width: 14.4ex;
                    padding: 5px;
                    }
                </string>
   </property>
   <property name="placeholderText">
    <string>Notes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="launch_url">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>230</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gill Sans</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
                    border: 2px solid #696969;
                    border-radius: 6px;
                    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                    stop: 0 #f6f7fa, stop: 1 #dadbde);
                    min-width: 80px;
                    }

                    QPushButton:pressed {
                    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                    stop: 0 #dadbde, stop: 1 #f6f7fa);
                    }

                    QPushButton:selected, QPushButton:hover {
                    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                    stop: 0 #484848, stop: 0.4 #585858,
                    stop: 0.5 #585858, stop: 1.0 #484848);
                    }

                    QPushButton {
                    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                    stop: 0 #686868, stop: 0.4 #505050,
                    stop: 0.5 #686868, stop: 1.0 #505050);
                    border: 2px solid #303030;
                    border-bottom-color: #303030; /* same as the pane color */
                    border-top-left-radius: 3px;
                    border-top-right-radius: 3px;
                    min-width: 14.4ex;
                    padding: 7px;
                    }

                    QPushButton:flat {
                    border: none; /* no border for a flat push button */
                    }

                    QPushButton:default {
                    border-color: navy; /* make the default button prominent */
                    }

                    QPushButton:selected {
                    border-color: #FF0000;
                    border-bottom-color: #FF0000; /* same as pane color */
                    }
                </string>
   </property>
   <property name="text">
    <string>Launch URL</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>end_time</tabstop>
  <tabstop>meeting_url</tabstop>
  <tabstop>meeting_name</tabstop>
  <tabstop>start_time</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
